src/screens/transaction-details/TransactionDetails.tsx

```diff
@@ -11,15 +11,22 @@ import { View } from 'react-native';
 import { BackgroundProvider } from '@/design-system';
 import { useSelector } from 'react-redux';
 import { AppState } from '@/redux/store';
+import { StackNavigationProp } from '@react-navigation/stack';
 
-type Params = {
-  transaction: RainbowTransaction;
+type RouteParams = {
+  TransactionDetails: {
+    transaction: RainbowTransaction;
+  };
 };
 
-export const TransactionDetails: React.FC = () => {
-  const route = useRoute();
+type Props = {
+  route: RouteProp<RouteParams, 'TransactionDetails'>;
+  navigation: StackNavigationProp<RouteParams, 'TransactionDetails'>;
```

Comment:
maybe i am missing something but is the `navigation` attribute actually used?

Reply:
Fixed that!